[2025-10-05T22:34:03.546+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flujo_RAG.conectar_minio manual__2025-10-05T22:33:57+00:00 [queued]>
[2025-10-05T22:34:03.549+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flujo_RAG.conectar_minio manual__2025-10-05T22:33:57+00:00 [queued]>
[2025-10-05T22:34:03.549+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-10-05T22:34:03.554+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): conectar_minio> on 2025-10-05 22:33:57+00:00
[2025-10-05T22:34:03.561+0000] {standard_task_runner.py:57} INFO - Started process 215 to run task
[2025-10-05T22:34:03.565+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'flujo_RAG', 'conectar_minio', 'manual__2025-10-05T22:33:57+00:00', '--job-id', '159', '--raw', '--subdir', 'DAGS_FOLDER/RAG_dag.py', '--cfg-path', '/tmp/tmpeuj061i3']
[2025-10-05T22:34:03.568+0000] {standard_task_runner.py:85} INFO - Job 159: Subtask conectar_minio
[2025-10-05T22:34:03.595+0000] {task_command.py:415} INFO - Running <TaskInstance: flujo_RAG.conectar_minio manual__2025-10-05T22:33:57+00:00 [running]> on host c16a2a509134
[2025-10-05T22:34:03.632+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flujo_RAG' AIRFLOW_CTX_TASK_ID='conectar_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-10-05T22:33:57+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-05T22:33:57+00:00'
[2025-10-05T22:34:03.634+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-10-05T22:34:03.634+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-10-05T22:34:03.635+0000] {warnings.py:109} WARNING - <string>:8: AirflowProviderDeprecationWarning: AWS Connection (conn_id='minio_s3', conn_type='s3') has connection type 's3', which has been replaced by connection type 'aws'. Please update your connection to have `conn_type='aws'`.

[2025-10-05T22:34:03.635+0000] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='minio_s3', conn_type='s3') credentials retrieved from login and password.
[2025-10-05T22:34:03.635+0000] {warnings.py:109} WARNING - <string>:8: AirflowProviderDeprecationWarning: Host minio specified in the connection is not used. Please, set it on extra['endpoint_url'] instead

[2025-10-05T22:34:03.866+0000] {logging_mixin.py:151} INFO - âœ… El bucket 'respaldo2' ya existe.
[2025-10-05T22:34:03.887+0000] {logging_mixin.py:151} INFO - ðŸ“„ Archivo 'prueba.txt' subido a bucket 'respaldo2'.
[2025-10-05T22:34:03.888+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-10-05T22:34:03.893+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=flujo_RAG, task_id=conectar_minio, execution_date=20251005T223357, start_date=20251005T223403, end_date=20251005T223403
[2025-10-05T22:34:03.908+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-10-05T22:34:03.924+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
